src/webapp/src/templates/components/hint.peb

```diff
@@ -1,3 +1,4 @@
+/**
```

Comment:
Why is this here Is there a **/ end to the comment somewhere This feels like it was accidentally commented out?

Reply:
Fixed.